### DISCUSSION TITLE
FreeCAD importer

### DISCUSSION THREAD

**Discussion by ebrahimraeyat** [xcfem/xc#81] [Q&A]
Hi All. I tried to create a model in FreeCAD and get the block of points and  lines  in .py file. I set the name of the object in FreeCAD with IFCSoleh, but  it gives me error, I trace the base class, reader_base.py and the `nameToImport` method return True:

![image](https://user-images.githubusercontent.com/8196112/107883307-76951980-6f03-11eb-8e78-26b683ccc29f.png)

![image](https://user-images.githubusercontent.com/8196112/107883455-49953680-6f04-11eb-8948-51882fa9fe6b.png)

FreeCAD file:
[soleh.zip](https://github.com/xcfem/xc/files/5978100/soleh.zip)

**Comment by lcpt** [comment]
Hi, @ebrahimraeyat 

Sorry for my late reply. I'm quite busy these days.

There is something to import inside IFCSoleh? I think the warning is related to an empty object group (but I'm not sure). Could you take a look inside the freecad_reader module?

I'll try to take a look into this next weekend.
